src/saturation/Saturation.c

```diff
@@ -0,0 +1,140 @@
+#include "Saturation.h"
+#include <math.h>
+
+double max(double x, double y);
+double min(double x, double y);
+double Hue(double red, double green, double blue, double cmax, double delta);
+double absolu(double x);
+void Saturation(Image* Bitmap, int x, int y,double sat);
+int Cap(double x);
+double CapSat(double x);
+
+void SaturationAbsolue(Image* Bitmap, double Indice)
+{
+  for(double x = 0; x < Bitmap->height; ++x)
+    {
+      for(double y = 0; y < Bitmap->width; ++y)
+	{
+	  Saturation(Bitmap, y, x, Indice);
+	}
+    }
+
+}
+
+
+void Saturation(Image* Bitmap, int x, int y,double sat)
```

Comment:
Use `if else` instead of chained `if` to make it faster.
This function is called for every pixel of the image so it can have a significant impact.